crates/unleash-edge-feature-refresh/src/lib.rs

```diff
@@ -154,21 +154,6 @@ pub struct FeatureRefresher {
     pub client_meta_information: ClientMetaInformation,
 }
 
-impl Default for FeatureRefresher {
```

Comment:
This isn't safe to default. Real world usage requires the arcs here to be shared because other things populate the state here. The only place we use that is currently in test code so I'm taking it away before it does become a prod dependency